src/Calculator/EquationStylePanelControl.xaml

```diff
@@ -80,6 +80,7 @@
                              Fill="{x:Bind}"
                              StrokeThickness="0"
                              AutomationProperties.Name="{x:Bind local:EquationStylePanelControl.GetColorAutomationName((Brush))}"
+                             ToolTipService.ToolTip="{x:Bind local:EquationStylePanelControl.GetColorAutomationName((Brush))}"
```

Comment:
All of the colors show up as black
![image](https://user-images.githubusercontent.com/6334170/79266400-5e202280-7e4c-11ea-8672-1dedb850233b.png)